crates/uv-dev/src/fetch_python.rs

```diff
@@ -1,6 +1,7 @@
 use fs_err as fs;
 use std::collections::HashMap;
-use std::{path::PathBuf, str::FromStr};
+
```

Comment:
nit: The import formatting is different from what we usually do

Reply:
Is there a way we can automate this? This is just how rust-analyzer imports the names.